apps/emqx/src/emqx_schema.erl

```diff
@@ -1488,27 +1675,57 @@ base_listener() ->
         {"bind",
             sc(
                 hoconsc:union([ip_port(), integer()]),
-                #{required => true}
+                #{
+                    required => true,
+                    desc => "IP address and port for the listening socket."
+                }
             )},
         {"acceptors",
             sc(
                 integer(),
-                #{default => 16}
+                #{
+                    default => 16,
+                    desc => "The size of the listener's receiving pool."
```

Comment:
nit, acceptors pool.

I guess it is copied from somewhere?

Reply:
Yes, indeed. Will fix in the next PR.